<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HCalc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="historyListWidget"/>
      </item>
      <item>
       <widget class="QLineEdit" name="inputLine">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Input here...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QPushButton" name="nineButton">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="sevenButton">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="subtractionButton">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="eightButton">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="zeroButton">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="oneButton">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="twoButton">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="sixButton">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="threeButton">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="fourButton">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="fiveButton">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="additionButton">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="multiplicationButton">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="leftParenthesisButton">
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="rightParenthesisButton">
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="divisionButton">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="equalsbutton">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="actionGenerate_Tokens"/>
    <addaction name="actionGenerate_Postfix"/>
    <addaction name="actionRun_Tests"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionGenerate_Tokens">
   <property name="text">
    <string>&amp;Generate Tokens</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionGenerate_Infix">
   <property name="text">
    <string>Generate Infix</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionGenerate_Postfix">
   <property name="text">
    <string>Generate &amp;Postfix</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRun_Tests">
   <property name="text">
    <string>&amp;Run Tests</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>inputLine</sender>
   <signal>returnPressed()</signal>
   <receiver>equalsbutton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
